ip-range-matcher/src/com/dataiku/dip/plugins/iprangematcher/IPRangeMatcher.java

```diff
@@ -0,0 +1,278 @@
+package com.dataiku.dip.plugins.iprangematcher;
+
+import com.dataiku.dip.datalayer.Column;
+import com.dataiku.dip.datalayer.Processor;
+import com.dataiku.dip.datalayer.Row;
+import com.dataiku.dip.datalayer.SingleRowProcessor;
+
+import com.dataiku.dip.shaker.model.StepParams;
+import com.dataiku.dip.shaker.processors.Category;
+import com.dataiku.dip.shaker.processors.ProcessorMeta;
+import com.dataiku.dip.shaker.processors.ProcessorTag;
+import com.dataiku.dip.shaker.server.ProcessorDesc;
+import com.dataiku.dip.util.ParamDesc;
+
+import com.dataiku.dip.utils.ErrorContext;
+import com.google.common.collect.Sets;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class IPRangeMatcher extends SingleRowProcessor implements Processor {
+    public static final Pattern RANGE_PATTERN = Pattern.compile("(\\d+)\\.(\\d+)\\.(\\d+)\\.(\\d+)[-_\\t ]{1,3}(\\d+)\\.(\\d+)\\.(\\d+)\\.(\\d+)");
```

Comment:
Without going to regex-matching exactly 0 to 255, I'd replace `\d` by `[0-2]\d{0,2}` (or at least `\d{1,3}`).